### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by abrormukimov**
# In this milestone:

- I added multiple pages

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,35 @@
...
+        )}
+        exact
+      />
+      <Route
+        path="/Home"
+        element={
+          <Home />
+        }
+        exact
+      />
```
Comment (julie-ify) [inline src/App.js:22]: - [ ] Let's make the `<Home />` to have the root path route `/` and `<Calculator />` to have `/Calculator` path as the home page is always the root path. Also, it's a requirement. See suggestion:

```suggestion
    <Routes>
      <Route
        path="/"
        element={(
           <Home />
        )}
        exact
      />
      <Route
        path="/Calculator"
        element={
          <Calculator />
        }
        exact
      />
```

File: src/components/Calculator.js [hunk 2]
```diff
@@ -5,25 +5,30 @@ import SimpleOperations from './Buttons/SimpleOperations';
...
+    <div>
+      <Header />
+      <Popup />
+      <div className={classes.container}>
+        <Input value={currentInnerText} />
+        <CalculatorOperations onclick={changeHandler} />
+        <Numbers onclick={changeHandler} />
+        <SimpleOperations onclick={changeHandler} />
+      </div>
     </div>
```
Comment (julie-ify) [inline src/components/Calculator.js:30]: - [ ] Let's add the left text to our Calculator page so as to match the wireframe.

 Your Implementation | Wireframe
:---:|:---:
 |

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @abrormukimov,

Good job so far!

## Highlight!
- Good feature branch name ✔️ 
- No linter errors ✔️  
- Good code organization and working app ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
Hi @abrormukimov 

Wooow👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

##  Optional suggestion

- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**